web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -12,6 +12,8 @@
         <script src="<c:url value="/resources/js/atsy-statehistory-create.js" />"></script>
         <script src="<c:url value="/resources/thirdparty/bootstrap-validator/validator.js" />"
                         type="text/javascript"></script>
+        <script src="<c:url value="/resources/thirdparty/bootstrap-datepicker/bootstrap-datepicker.js" />" type="text/javascript"></script>
+        <link href="<c:url value="/resources/thirdparty/bootstrap-datepicker/bootstrap-datepicker.css" />" rel="stylesheet" type="text/css">
```

Comment:
Extract `<c:url>` tag from the `<link>` tag. Only make this change for the code you added.